content/documentation/nodes/matrix/_index.md

```diff
@@ -6,4 +6,14 @@ chapter : true
 
 # Matrix
 
-This is the reference for matrix nodes in Animation Nodes.
+### Data Types
+
+- **Matrix** - A matrix. It stores location, rotation and scale.
```

Comment:
A transformation matrix. It can represent a transformation or the transforms of an object, that is, its location, rotation, and scale.

Reply:
Okay.